### PR TITLE
renamed the handler

### PR DESCRIPTION
**Pull request by da1nerd**
@richmahn I've cloned the original door43-job-handler repo and did a first pass to rename the handler in the code. I'd like to get a set of eyes on this to make sure I've renamed things correctly before I continue modifying this.

I suspect in reality the naming is not as critical, but I'd like to stick with the expected convention.

### CODE REVIEW

File: webhook.py [hunk 1]
```diff
@@ -78,7 +78,7 @@
 AppSettings(prefix=prefix)
 if prefix not in ('', 'dev-'):
     AppSettings.logger.critical(f"Unexpected prefix: '{prefix}' — expected '' or 'dev-'")
-door43_stats_prefix = f"door43.{'dev' if prefix else 'prod'}"
+door43_stats_prefix = f"door43-catalog.{'dev' if prefix else 'prod'}"
 job_handler_stats_prefix = f"{door43_stats_prefix}.job-handler"
```
Comment (da1nerd) [inline webhook.py:82]: I'm not certain if I should rename this to `door43-catalog....` or `...catalog-job-handler`.

File: callback.py [hunk 2]
```diff
@@ -38,7 +38,7 @@
 AppSettings(prefix=prefix)
 if prefix not in ('', 'dev-'):
     AppSettings.logger.critical(f"Unexpected prefix: '{prefix}' — expected '' or 'dev-'")
-door43_stats_prefix = f"door43.{'dev' if prefix else 'prod'}"
+door43_stats_prefix = f"door43-catalog.{'dev' if prefix else 'prod'}"
 job_handler_stats_prefix = f"{door43_stats_prefix}.job-handler" # Can't add .callback here coz we also have .total
```
Comment (da1nerd) [inline callback.py]: I'm not certain if I should rename this to `door43-catalog....` or `...catalog-job-handler`.

File: Dockerfile-developBranch [hunk 3]
```diff
@@ -9,7 +9,7 @@ WORKDIR /code
 RUN pip3 install --upgrade pip
 RUN pip3 install --requirement requirements.txt
 
-CMD [ "rq", "worker", "--config", "rq_settings", "--name", "D43_Dev_JobHandler" ]
```
Comment (da1nerd) [inline Dockerfile-developBranch:12]: I'm not sure if this should be `D43_Dev_CatalogJobHandler` or `D43Catalog_Dev_JobHandler`.
This question applies to the `Dockerfile-masterBranch` file as well.

File: webhook.py (hunk 1 above)
Comment (richmahn) [reply webhook.py:82]: That's fine how you have it.

File: Dockerfile-developBranch (hunk 3 above)
Comment (richmahn) [reply Dockerfile-developBranch:12]: It's just how the door43-enqueue-job repo will add the job to the stack, so whatever you want. D43_Dev_CatalogJobHandler should be fine. This is of course for development environment.

**Pull request merged**